x/posts/client/cli/cli_test.go

```diff
@@ -185,6 +187,7 @@ func (s *IntegrationTestSuite) TestCmdQueryPost() {
 					"cosmos13t6y2nnugtshwuy0zkrq287a95lyy8vzleaxmd",
 					0,
 					nil,
+					[]string{},
```

Comment:
Can we use `nil` instead of empty slice?

Reply:
Unfortunately, we can't. I don't know why but the test fails if we use `nil` instead of an empty slice while comparing the two posts (expected vs actual)

Reply:
Got it